React/wbooks/src/app/components/RouteHandler/index.js

```diff
@@ -2,15 +2,15 @@ import React, { Component } from 'react';
 import { Route } from 'react-router-dom';
 import { Redirect } from 'react-router';
 
-import localStorageService from '../../../services/localstorageService';
+import {retrieveUserFromLocalStorage} from '../../../services/localstorageService';
```

Comment:
i recommend use `import * as localStorageService from ...`  to use them later as `localStorageService.retrieveUserFromLocalStorage();`

Reply:
Remember add space between braces